addons/mail/models/mail_thread.py

```diff
@@ -2220,7 +2220,7 @@ def _notify_record_by_email(self, message, recipients_data, msg_vals=False,
 
         model = msg_vals.get('model') if msg_vals else message.model
         model_name = model_description or (self.with_lang().env['ir.model']._get(model).display_name if model else False) # one query for display name
-        recipients_groups_data = self._notify_classify_recipients(partners_data, model_name)
+        recipients_groups_data = self._notify_classify_recipients(partners_data, model_name, msg_vals=msg_vals)
```

Comment:
Here `msg_vals` is `False` sometimes so you may need to handle that case here to bring back runbot to green